libraries/rush-lib/src/logic/Autoinstaller.ts

```diff
@@ -24,13 +24,19 @@ import { LastInstallFlag } from '../api/LastInstallFlag';
 import { RushCommandLineParser } from '../cli/RushCommandLineParser';
 import type { PnpmPackageManager } from '../api/packageManager/PnpmPackageManager';
 
-interface IAutoinstallerOptions {
+/**
+ * @public
+ */
+export interface IAutoinstallerOptions {
   autoinstallerName: string;
   rushConfiguration: RushConfiguration;
   rushGlobalFolder: RushGlobalFolder;
   restrictConsoleOutput?: boolean;
 }
 
+/**
+ * @public
+ */
```

Comment:
Are these ready to be `@public` yet, or should they start out as `@beta`?

I don't remember if we ever did a real API review on these.